internal/backends/python/gen_pypi_map/package.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"regexp"
+	"strings"
+)
+
+var normalizationPattern = regexp.MustCompile(`[-_.]+`)
+
+// normalizePackageName implements NormalizePackageName for the Python
+// backends.
+// See https://packaging.python.org/en/latest/specifications/name-normalization/
+func normalizePackageName(name string) string {
+	name = strings.ToLower(name)
+	name = normalizationPattern.ReplaceAllString(name, "-")
+	return name
+}
```

Comment:
should probably share this code with the function that's already in `internal/backens/python/python.go`

Reply:
It makes me think back to the OSS request of sharing our Go python tooling as a public lib. Perhaps someday, but this is still too low of a threshold to warrant that at this time, imo. I'm definitely thinking in that direction though!